source/common/thread_local/thread_local_impl.cc

```diff
@@ -157,19 +159,20 @@ void InstanceImpl::runOnAllThreads(Event::PostCb cb) {
   cb();
 }
 
-void InstanceImpl::runOnAllThreads(Event::PostCb cb, Event::PostCb all_threads_complete_cb) {
+void InstanceImpl::runOnAllThreads(std::function<void()> cb,
```

Comment:
I think these two could be move-only? cb_guard is a single instance that can own cb, and all_threads_complete_cb is moved to the main thread.

Reply:
Yeah, makes sense.